spec/models/post_spec.rb

```diff
@@ -1,5 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe Post, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  before(:each) do
+    @user = User.create(name: 'John', bio: 'Jons Bio', posts_counter: 0)
+  end
+
+  it 'eager loads all files without errors' do
+    expect { Rails.application.eager_load! }.not_to raise_error
+  end
```

Comment:
Note:
This method is not related to the N+1 issue that we are talking about in the project requirements. `Rails.application.eager_load!` method eagerly loads the ruby files from disk to computer memory, in project requirement we are talking about getting values eagerly from the SQL database. You can check [this article](https://dev.to/junko911/rails-n-1-queries-and-eager-loading-10eh) for more information about the N+1 query issue

Reply:
Done! 👍